sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/samples/LocationSample.md

```diff
@@ -0,0 +1,19 @@
+# Location verify
+
+```C#
+using Azure.Communication.ProgrammableConnectivity;
+
+private readonly Uri _endpoint = new Uri("https://eastus.prod.apcgatewayapi.azure.com");
```

Comment:
I will make the number `+5000000000` to make it clear that it's a fake number. I want to use numbers since in the future we may have some validation.

I will make the endpoint `https://your-endpoint-here.com`